opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/wcaro/SideNavigationMenuTests.java

```diff
@@ -0,0 +1,113 @@
+package org.smartregister.chw.activity.wcaro;
+
+import android.Manifest;
+
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.smartregister.chw.activity.LoginActivity;
+import org.smartregister.chw.activity.utils.Constants;
+import org.smartregister.chw.activity.utils.Order;
+import org.smartregister.chw.activity.utils.OrderedRunner;
+import org.smartregister.chw.activity.utils.Utils;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withSubstring;
+
+@RunWith(OrderedRunner.class)
+public class SideNavigationMenuTests {
+
+    @Rule
+    public ActivityTestRule<LoginActivity> intentsTestRule = new ActivityTestRule<>(LoginActivity.class);
+
+    @Rule
+    public GrantPermissionRule permissionRule = GrantPermissionRule.grant(android.Manifest.permission.CALL_PHONE);
+
+    @Rule
+    public GrantPermissionRule mRuntimePermissionRule = GrantPermissionRule.grant(Manifest.permission.CAMERA);
+
+    @Rule
+    public GrantPermissionRule mRuntimePermissionRule1 = GrantPermissionRule.grant(Manifest.permission.ACCESS_FINE_LOCATION);
+
+    private Utils utils = new Utils();
+
+    public ActivityTestRule<LoginActivity> mActivityTestRule = new ActivityTestRule<>(LoginActivity.class);
+
+    @Before
+    public void setUp() throws InterruptedException {
+        utils.logIn(Constants.WcaroConfigUtils.wCaro_username, Constants.WcaroConfigUtils.wCaro_password);
+        utils.openDrawer();
+    }
+
+    @Test
+    @Order(order = 1)
+    public void leftDrawerMenuResponsiveness() throws InterruptedException{
+        Thread.sleep(500);
+        onView(withSubstring("Registers"))
+                .check(matches(isDisplayed()));
+    }
+
+    @Test
+    @Order(order = 2)
+    public void correctAppNameTest() throws InterruptedException{
+        Thread.sleep(500);
+        onView(withSubstring(Constants.WcaroConfigUtils.appName))
+                .check(matches(isDisplayed()));
+    }
+
+    @Test
+    @Order(order = 3)
+    public void correctRegistersTest() throws InterruptedException{
+        Thread.sleep(500);
+        onView(withSubstring(Constants.GenericConfigUtils.anc))
+                .check(matches(isDisplayed()));
+        onView(withSubstring(Constants.GenericConfigUtils.pnc))
+                .check(matches(isDisplayed()));
+        onView(withSubstring(Constants.GenericConfigUtils.child))
+                .check(matches(isDisplayed()));
+    }
+
+    @Test
+    @Order(order = 4)
+    public void syncTest() throws InterruptedException{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)